### PR TITLE
documentation added for multi-container implementation

### PR DESCRIPTION
**Pull request by AadilLatif**

### CODE REVIEW

**Comment by josephmckinsey** [comment]
Can you add the sphinx-youtube requirement to the docs/requirements.txt?

**Comment by AadilLatif** [comment]
> Can you add the sphinx-youtube requirement to the docs/requirements.txt?

Done

File: docs/images/step_4.svg
Comment (josephmckinsey) [inline docs/images/step_4.svg:1]: There's some small overlap on the text, here, but I hope it's just a github thing.

File: docs/sgidal-example.code-workspace
Comment (josephmckinsey) [inline docs/sgidal-example.code-workspace:1]: Can we delete this file?

**Review by josephmckinsey — CHANGES_REQUESTED** [review]

**Review by josephmckinsey — APPROVED** [review]

**Pull request merged**